Docs/source/install/batch/pjm.rst

```diff
@@ -0,0 +1,45 @@
+.. note::
+
+   This section is a stub and improvements to complete the ``(TODO)`` sections are welcome.
```

Comment:
@lucafedeli88 do you have further PJM references that we could use to fill the TODOs in follow-up PRs?